### PR TITLE
Correlations bug fix

### PR DESCRIPTION
**Pull request by shadrackkibet**
This fixes some of the issues raised by @lilyclements at #3657. I am yet to address other issues as well.

### CODE REVIEW

**Comment by maxwellfundi** [comment]
@shadrackkibet  could you please list down other issues here that needs to be addressed? We need this merged soon for a version we are demonstrating soon

**Comment by shadrackkibet** [comment]
issue with this now;
Plots not displaying on the output window.i am using icallType 3  but it displays a blank grid.

**Comment by shadrackkibet** [comment]
@dannyparsons  this dialog is now working fine. This is ready for you.

**Comment by dannyparsons** [comment]
@lilyclements is this ready now?

**Comment by lilyclements** [comment]
@dannyparsons yes this is ready

File: instat/dlgCorrelation.vb [hunk 1]
```diff
@@ -122,38 +126,44 @@ Public Class dlgCorrelation
...
         clsRGraphicsFuction.SetRCommand("ggpairs")
         clsRGraphicsFuction.iCallType = 3
+        clsRGraphicsFuction.bExcludeAssignedFunctionOutput = False
         clsRGraphicsFuction.AddParameter("data", clsRFunctionParameter:=clsTempFunc)
 
         clsRGGscatMatrixFunction.SetPackageName("GGally")
         clsRGGscatMatrixFunction.SetRCommand("ggscatmat")
         clsRGGscatMatrixFunction.iCallType = 3
+        clsRGGscatMatrixFunction.bExcludeAssignedFunctionOutput = False
         clsRGGscatMatrixFunction.AddParameter("data", clsRFunctionParameter:=clsTempFunc)
```
Comment (dannyparsons) [inline instat/dlgCorrelation.vb]: This shouldn't be done manually, there's no reason why the selector isn't linked to these functions? Happy that this works for now.

File: instat/dlgCorrelation.vb [hunk 2]
```diff
@@ -235,10 +246,23 @@ Public Class dlgCorrelation
...
+            'Else ' in future we'll have to add these back and have whatever they checked as checked here.
+        End If
     End Sub
 
     Private Sub ucrReceiverFirstColumn_ControlContentsChanged(ucrChangedControl As ucrCore) Handles ucrReceiverFirstColumn.ControlContentsChanged, ucrReceiverSecondColumn.ControlContentsChanged, ucrReceiverMultipleColumns.ControlContentsChanged, ucrPnlColumns.ControlContentsChanged, ucrPnlCompletePairwise.ControlContentsChanged, ucrPnlMethod.ControlContentsChanged
         TestOKEnabled()
     End Sub
+
+    Private Sub ucrSelectorCorrelation_ControlContentsChanged(ucrChangedControl As ucrCore) Handles ucrSelectorCorrelation.ControlContentsChanged
+        clsTempFunc = ucrSelectorCorrelation.ucrAvailableDataFrames.clsCurrDataFrame
```
Comment (dannyparsons) [inline instat/dlgCorrelation.vb]: This isn't needed, even with the current implementation. The function changes automatically in the selector.

**Pull request merged**